.github/workflows/build_and_test.yml

```diff
@@ -88,40 +89,51 @@ jobs:
         env:
           DOCKER_BUILDKIT: 1
         run: >
-          docker build -t integritee-cli-client-${{ matrix.flavor_id }}-${{ github.sha }}
+          docker build -t integritee-cli-client-${{ env.IMAGE_SUFFIX }}
           --target deployed-client
           --build-arg WORKER_MODE_ARG=${{ matrix.mode }} --build-arg ADDITIONAL_FEATURES_ARG=${{ matrix.additional_features }}
           -f build.Dockerfile .
 
       - run: docker images --all
 
       - name: Test Enclave # cargo test is not supported in the enclave, see: https://github.com/apache/incubator-teaclave-sgx-sdk/issues/232
-        run: docker run ${{ env.DOCKER_DEVICES }} ${{ env.DOCKER_VOLUMES }} integritee-worker-${{ matrix.flavor_id }}-${{ github.sha }} test --all
+        run: docker run ${{ env.DOCKER_DEVICES }} ${{ env.DOCKER_VOLUMES }} integritee-worker-${{ env.IMAGE_SUFFIX }} test --all
 
       - name: Export worker image(s)
         run: |
-          docker image save integritee-worker-${{ matrix.flavor_id }}-${{ github.sha }} | gzip > integritee-worker-${{ matrix.flavor_id }}-${{ github.sha }}.tar.gz
-          docker image save integritee-cli-client-${{ matrix.flavor_id }}-${{ github.sha }} | gzip > integritee-cli-client-${{ matrix.flavor_id }}-${{ github.sha }}.tar.gz
+          docker image save integritee-worker-${{ env.IMAGE_SUFFIX }} | gzip > integritee-worker-${{ env.IMAGE_SUFFIX }}.tar.gz
+          docker image save integritee-cli-client-${{ env.IMAGE_SUFFIX }} | gzip > integritee-cli-client-${{ env.IMAGE_SUFFIX }}.tar.gz
 
       - name: Upload worker image
         uses: actions/upload-artifact@v3
         with:
-          name: integritee-worker-${{ matrix.flavor_id }}-${{ github.sha }}.tar.gz
-          path: integritee-worker-${{ matrix.flavor_id }}-${{ github.sha }}.tar.gz
+          name: integritee-worker-${{ env.IMAGE_SUFFIX }}.tar.gz
+          path: integritee-worker-${{ env.IMAGE_SUFFIX }}.tar.gz
 
       - name: Upload CLI client image
         uses: actions/upload-artifact@v3
         with:
-          name: integritee-cli-client-${{ matrix.flavor_id }}-${{ github.sha }}.tar.gz
-          path: integritee-cli-client-${{ matrix.flavor_id }}-${{ github.sha }}.tar.gz
+          name: integritee-cli-client-${{ env.IMAGE_SUFFIX }}.tar.gz
+          path: integritee-cli-client-${{ env.IMAGE_SUFFIX }}.tar.gz
+
+      - name: Create Enclave Digest File
+        run: |
+          mrenclave_hex=$(make mrenclave | grep -o -E '[0-9a-fA-F]{64}')
+          echo "$mrenclave_hex" > mrenclave-${{ env.IMAGE_SUFFIX }}.hex
```

Comment:
So this is the code that essentially needs to be run to create an mrenclave file. Obviously, it doesn't work just like that because our `make mrenclave` assumes an enclave binary at `./bin/enclave.so`. We don't have access to the binaries here, as we build everything in docker. I suppose the best way to solve this is to create an entry point in the docker image such that we can run `docker run <image> mrenclave`?

What do you think about that? @mosonyi, @brenzi?

Reply:
Hmm, it fails indeed at this step currently in the CI, but not where I expected:

```
  mrenclave_hex=$(make mrenclave | grep -o -E '[0-9a-fA-F]{64}')
  echo "$mrenclave_hex" > mrenclave--offchain-worker-d86b797d032c035a1fc7d0a343c0832af32d0564.hex
  shell: /usr/bin/bash -e {0}
  env:
    CARGO_TERM_COLOR: always
    LOG_DIR: logs
    BUILD_CONTAINER_NAME: integritee_worker_enclave_test
    FINGERPRINT: 31664
    IMAGE_SUFFIX: -offchain-worker-d86b797d032c035a1fc7d0a343c0832af32d0564
    DOCKER_DEVICES: --device=/dev/sgx/enclave --device=/dev/sgx/provision
    DOCKER_VOLUMES: --volume /var/run/aesmd:/var/run/aesmd --volume /etc/sgx_default_qcnl.conf:/etc/sgx_default_qcnl.conf
/bin/sh: 1: file: not found
/bin/sh: 1: /opt/intel/sgxsdk/bin/x64/sgx_sign: not found
```

So the follow up question would be: Is `/opt/intel/sgxsdk/bin/x64/sgx_sign` available in the the worker image we build?

Reply:
Using a docker command now.